main/streams/filter.c

```diff
@@ -192,11 +192,17 @@ PHPAPI void php_stream_bucket_append(php_stream_bucket_brigade *brigade, php_str
 PHPAPI void php_stream_bucket_unlink(php_stream_bucket *bucket)
 {
 	if (bucket->prev) {
+		if (bucket->prev->next != bucket) {
```

Comment:
```suggestion
		if (UNEXPECTED(bucket->prev->next != bucket)) {
```
or we could maybe introduce macro similar to ZEND_MM_CHECK that could be in other places as well..

Reply:
Is it worth adding a macro for two usages?